### PR TITLE
Update auth0-forward information

### PR DESCRIPTION
**Pull request by tamigoodall**
Added sentence about proxy server IP address added to whitelist.

### CODE REVIEW

**Review by sebadoom — DISMISSED** [review]

File: articles/api-auth/tutorials/using-resource-owner-password-from-server-side.md [hunk 1]
```diff
@@ -63,13 +63,21 @@ Due to security considerations, the configuration stated on Step 3 will not be a
 
 ### Sending the end-user IP from your server
 
-To send the end-user IP from your server, include a `auth0-forwarded-for` header with the value of the end-user IP address. If the IP is valid, Auth0 will use it as the source IP for brute-force protection. It is important to make sure the provided IP address really belongs to your end user.
+To send the end-user IP from your server, include a `auth0-forwarded-for` header with the value of the end-user IP address. If the IP is valid, Auth0 will use it as the source IP for [brute-force protection](/anomaly-detection). It is important to make sure the provided IP address really belongs to your end user.
```
Comment (sebadoom) [inline articles/api-auth/tutorials/using-resource-owner-password-from-server-side.md]: `If the IP is valid` -> I'd change this to `If the "auth0-forwarded-for" header is marked as trusted, as explained above, Auth0 will use....`

Comment (tamigoodall) [reply articles/api-auth/tutorials/using-resource-owner-password-from-server-side.md]: Updated

**Review by sebadoom — DISMISSED** [review]

**Review by sebadoom — DISMISSED** [review]

File: articles/api-auth/tutorials/using-resource-owner-password-from-server-side.md [hunk 2]
```diff
@@ -63,13 +63,21 @@ Due to security considerations, the configuration stated on Step 3 will not be a
...
+When using the resource owner password grant from your webserver with brute-force protection enabled, you could specify a whitelist of IPs that will not be considered when triggering brute-force protection. Both the `auth0-forwarded-for` IP address and the IP address of the proxy server will be taken into account for IP address whitelists. 
+
 
 ::: warning
 <strong>Warning!</strong> Trusting headers like the <code>x-forwarded-for</code> (or, in general, data from application) as source for the end-user IP can be a big risk. This should not be done unless you know you can trust that header, since it is easy to spoof and makes possible to bypass the anomaly-detection validation.
 </div>
 :::
 
+::: note
+As of December 18, 2018, both the IP address passed as part of the `auth0-forwarded-for` header, and the IP address of the request itself, will be matched against the configured whitelist for brute-force protection.
```
Comment (aaguiarz) [inline articles/api-auth/tutorials/using-resource-owner-password-from-server-side.md]:
```suggestion
Both the IP address passed as part of the `auth0-forwarded-for` header, and the IP address of the request itself, will be matched against the configured whitelist for brute-force protection. 
```

File: articles/api-auth/tutorials/using-resource-owner-password-from-server-side.md [hunk 3]
```diff
@@ -63,13 +63,21 @@ Due to security considerations, the configuration stated on Step 3 will not be a
...
+
 
 ::: warning
 <strong>Warning!</strong> Trusting headers like the <code>x-forwarded-for</code> (or, in general, data from application) as source for the end-user IP can be a big risk. This should not be done unless you know you can trust that header, since it is easy to spoof and makes possible to bypass the anomaly-detection validation.
 </div>
 :::
 
+::: note
+As of December 18, 2018, both the IP address passed as part of the `auth0-forwarded-for` header, and the IP address of the request itself, will be matched against the configured whitelist for brute-force protection. 
+The IP address in the header will only be used if it is marked as trusted for the connections. This change only extends the whitelist functionality to the `auth0-forwarded-for` header IP address without removing previous functionality.
```
Comment (aaguiarz) [inline articles/api-auth/tutorials/using-resource-owner-password-from-server-side.md]:
```suggestion
The IP address in the header will only be used if it is marked as trusted for the connections.
```

**Review by jeffreylees — APPROVED** [review]

**Pull request merged**